docs/enterprise-solutions/sso-setup.mdx

```diff
@@ -0,0 +1,65 @@
+---
+title: "SSO Setup"
+sidebarTitle: "SSO Setup"
+description: "Configure Single Sign-On (SSO) for Cline Enterprise via WorkOS AuthKit."
+---
+
+## Overview
+Cline Enterprise integrates with your identity provider (IdP) via **WorkOS AuthKit** for SSO.
+
+This page describes, at a high level, how SSO is set up for Cline Enterprise using WorkOS AuthKit.
+
+If you haven’t completed initial onboarding, start with [Onboarding](/enterprise-solutions/onboarding).
+
+## Where setup happens
+SSO setup spans three places:
+
+1) **Cline Dashboard (app.cline.bot)**
+- Where you sign in and verify SSO works for your organization.
+
+2) **WorkOS dashboard**
+- Where your IdP connection is configured (AuthKit → Connections).
+
+3) **Your Identity Provider (Okta / Entra / Google Workspace / Keycloak, etc.)**
+- Where users, groups, and login policy are managed.
+
+<Info>
+This page intentionally avoids backend deployment/internal configuration details. If you’re onboarding to Cline Enterprise and need deployment-specific configuration values (like redirect URLs), contact your onboarding/support channel.
+</Info>
+
+## Using the Cline Dashboard
+Use the Cline Dashboard at https://app.cline.bot to:
+- complete sign-in and onboarding flows
+- verify users can authenticate via SSO
+
+<Info>
+The dashboard includes SSO-related callback routes used during authentication (for example, `/auth/callback` and `/auth/sso-callback`).
+</Info>
+
+## Configure your IdP connection in WorkOS
```

Comment:
I may be misunderstanding the intended audience for this documentation, but it seems we first need to explain how they’ll access their WorkOS dashboard. That process involves us sending an email with a link to their WorkOS configuration page to the designated admin.